app/views/organizations/edit.html.erb

```diff
@@ -47,6 +47,9 @@
                   <span class="input-group-text"><i class="fa fa-desktop"></i></span>
                   <%= f.input_field :short_name, disabled: true, class: "form-control" %>
                 <% end %>
+                <%= f.input :ndbn_member, label: 'NDBN Membership', wrapper: :input_group do %>
+                  <%= f.association :ndbn_member, label_method: :full_name, value_method: :id, label: false %>
```

Comment:
This will put every single member ID in the database into the select form, right? This is fine if there are maybe a thousand or so. Any more than that and it's going to impact performance (both network and rendering time) and typeahead would make more sense.

Reply:
@dorner good point. I don't think we'll see that many based on the current number of NDBN users. Right now it is 330. I think we can wait to see if performance slows down and then make that change.

TBH, I didn't add typeahead because I didn't find a clear-cut solution without using JS or adding a more complex system.